NGCHM/WebContent/javascript/UI-Manager.js

```diff
@@ -1273,6 +1272,7 @@
 			case 'ArrowLeft': // left key
 				if (document.activeElement.id !== "search_text"){
 					e.preventDefault();
+					if (!mapItem) return;
```

Comment:
"mapItem" is always either =primaryMap or null ?  If that's true, I suggest naming the variable primaryMap.